repl/src/main/scala/ammonite/repl/Util.scala

```diff
@@ -150,6 +155,7 @@ object Ex{
 
 object Util{
   type IvyMap = Map[(String, String, String), Set[String]]
+  type CompileCache = (Traversable[(String, Array[Byte])], Seq[ImportData])
```

Comment:
- Can we pull `Traversable[(String, Array[Byte])]` out into its own type alias? We use it 9 times in the codebase.